From b243117dcb27d7afea2a7808163cc6723b4cac48 Mon Sep 17 00:00:00 2001
From: Dennis Theisen <dennis@jadedpixel.com>
Date: Fri, 16 Oct 2009 19:24:45 +0200
Subject: [PATCH] Add a Beep tone when rake task fails because of pending migrations.
   * Background: When running e.g. tests that take a few minutes, most users use rake db:migrate and then do something else.
                 Because the console is in the background a default error/warn sound could help notify the user that there's a problem, so no time is wasted.

---
 railties/lib/rails/tasks/databases.rake |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/railties/lib/rails/tasks/databases.rake b/railties/lib/rails/tasks/databases.rake
index ed015e7..1671121 100644
--- a/railties/lib/rails/tasks/databases.rake
+++ b/railties/lib/rails/tasks/databases.rake
@@ -228,7 +228,7 @@ namespace :db do
         pending_migrations.each do |pending_migration|
           puts '  %4d %s' % [pending_migration.version, pending_migration.name]
         end
-        abort %{Run "rake db:migrate" to update your database then try again.}
+        abort %{Run "rake db:migrate" to update your database then try again.\a}
       end
     end
   end
-- 
1.6.3.1

